MeshLib/MeshSubset.h

```diff
@@ -91,6 +103,25 @@ class MeshSubset
         return _msh.getElements().cend();
     }
 
+    MeshSubset*
+    setIntersectionByNodes(std::vector<Node*> const& nodes) const
```

Comment:
the function name is confusing because it does not set anything to this object. I suggest something like `getIntersectionByNodes()`